### ISSUE TITLE
Page/project creation: having "Select all" and "Select None" buttons before the checkboxes is confusing for screen reader users

### ISSUE THREAD

**Issue by sepidehshahi** [accessibility-in-action/platform#835]
When filling out the pages for organization and project, screen reader user was confused by hearing the select all and select none buttons before the check boxes. They had no idea what those options are and they had to select all or none. It was recommended to move the buttons to the end of the list.

**Comment by greatislander** [comment]
@chosww Here's an example of this:

https://github.com/accessibility-exchange/platform/blob/64fe57ce1fab94234a2054e8c552e3e601b860e5/resources/views/projects/edit/1.blade.php#L31-L38

Basically, lines 33-36 should be moved after 33-37. There are a few cases where this component is used— you should be able to search for `enhancedCheckboxes()` to find them.
